main.py

```diff
@@ -1,121 +0,0 @@
-from flask import Flask, render_template, redirect, url_for, request
-from flask_bootstrap import Bootstrap5
```

Comment:
Dividing this file into smaller files is a very good idea! 
Code is much easier to analyze. This project is quite small, and all files are quite ok. I assume that you want to achieve something more than ok.
 What do you think about extracting business logic? For now you have everything connected. In commercial project it is very big problem when everything is connected with everything. We try to make layers .
I know you have started to make tests, so you can extract business logic later :) It is much easier to make refactor when project has tests.

I could give you example what I am writing about

@app.route("/edit", methods=["GET", "POST"])
def edit():
    from app.database import Movie, db

    movie_id = request.args.get("id")
    chosen_movie = MovieService(movie_id).get() 
    if chosen_movie is None:
       return 404.html // I am not sure how to write it in python
    with app.app_context():
        chosen_movie = db.get_or_404(Movie, movie_id)
        if request.method == "POST":
            chosen_movie.update( {rating=request.form["rating"], review=request.form["review"] )
            return redirect(url_for("home"))
    return render_template("edit.html", title=chosen_movie.title)

Reply:
for now, I have the MovieService module responsible for calling API - do you think it is a good idea to have the same module responsible for updating db? Or should I create a new one?